org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/util/JrtUtilTest.java

```diff
@@ -70,6 +74,19 @@ public void testGetNewJrtFileSystem() throws Exception {
 		assertSame(jrtSystem, jrtSystem3);
 	}
 
+	@Test
+	public void testWorksUnderInterruptCondition() throws IOException {
+		try (FileSystem fs = FileSystems.getFileSystem(URI.create("jrt:/"))) {
```

Comment:
java.lang.UnsupportedOperationException
	at java.base/jdk.internal.jrtfs.JrtFileSystem.close(JrtFileSystem.java:104)
	at org.eclipse.jdt.core.tests.compiler.util.JrtUtilTest.testWorksUnderInterruptCondition(JrtUtilTest.java:85)

Reply:
Fixed, even though it should only throw it for "the default file system" according to the javadoc...